JavaScript/a-for-await.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+async function * slowCounter() {
```

Comment:
We need consistent generator syntax. Do we use space before star? All other examples don't.